packages/webapp/src/components/Animals/Inventory/index.tsx

```diff
@@ -87,6 +91,7 @@ const PureAnimalInventory = ({
   const hasSearchResults = filteredInventory.length !== 0;
 
   const tableMaxHeight = !isDesktop || !containerHeight ? undefined : containerHeight - usedHeight;
+  const tableSpacerRowHeight = showActionMenu ? (isDesktop ? 96 : 120) : 0;
```

Comment:
I mostly agree I think it would make sense to incorporate it as part of the inventory layout vs table layout . But I don't think we can overlap content because it would make the bottom content unviewable.